docs/user-guide/pods.md

```diff
@@ -110,6 +110,24 @@ Pod is exposed as a primitive in order to facilitate:
 
 The current best practice for pets is to create a replication controller with `replicas` equal to `1` and a corresponding service. If you find this cumbersome, please comment on [issue #260](http://issue.k8s.io/260).
 
+
+## Termination of Pods
+
+Because pods represent running processes on nodes in the cluster, it is important to allow those processes to gracefully terminate when they are no longer needed (vs being violently killed with a KILL signal and having no chance to clean up). Users should be able to request deletion and know when processes terminate, but also be able to ensure that deletes eventually complete. Kubernetes represents a distributed system, so it's not always possible to observe graceful shutdown of a process, and in some cases processes may not cleanly shut down due to kernel or system issues. When a user requests deletion of a pod the system records the intended grace period before the pod is allowed to be forcefully killed, and a TERM signal is sent to the main process in each container. Once the grace period has expired the KILL signal is sent to those processes and the pod is then deleted from the API server. If processes from that pod are still running, the system will work to terminate them.
+
+An example flow:
+
+1. User sends command to delete Pod, with default grace period (30s)
+2. The Pod in the API server is updated with the time beyond which the Pod is considered "dead" along with the grace period.
+3. Pod shows up as "Terminating" when listed in client commands
+4. (simultaneous with 3) When the Kubelet sees that a Pod has been marked as terminating, the processes in the Pod are immediately sent the TERM signal.
```

Comment:
Clarify that the way the Kubelet sees that a pod has been marked as terminating, is that it sees that the time mentioned in step 2 has been set.